manifest/default.xml

```diff
@@ -60,6 +61,10 @@
 
   <project name="gocb" path="godeps/src/github.com/couchbase/gocb" remote="couchbase" revision="67fb8fa4c62451372108f31c93034c07d9d9f4ab"/>
 
+  <project name="gocbcore" path="godeps/src/gopkg.in/couchbase/gocbcore.v2" remote="couchbase" revision="22d72f05785ba26be033aa8b6726435b5e50a7a0"/>
+
+  <project name="jsonx" path="godeps/src/gopkg.in/couchbaselabs/jsonx.v1" remote="couchbaselabs" revision="5b7baa20429a46a5543ee259664cc86502738cad"/>
+
```

Comment:
I'm guessing, but probably gocb isn't being built directly but is referenced by some other tool. And I guess whatever portions of gocb are used don't import the gocbore (or gopkg.in) stuff.

I do notice that the gocb reference in branch-master.xml is on the 'server-4.6' branch, so maybe that's an earlier version of the code which didn't have those dependencies?